ibm/mas_devops/roles/sls_install/defaults/main.yml

```diff
@@ -7,6 +7,8 @@ sls_channel: 3.x
 # By default we will install into this namespace
 sls_namespace: ibm-sls
 
+sls_cfg_file: /tmp/slscfg-local-system.yaml
```

Comment:
Why has this been added/for what purpose?

We use a generated value for sls_cfg_file of `"{{ mas_config_dir }}/sls-{{ sls_instance_name | lower }}-{{ sls_namespace }}.yml"`

Reply:
We wanted to use this tmp file for configuration. But i see that it is getting generated. Removing it.